### PR TITLE
Adding Freeipa Full docs

### PR DESCRIPTION
**Pull request by kalinux**
Hi ,

I thinks its interesting docs for Freeipa and Gogs and will save a lot of time for another users.

Kindly Regards
Karen

### CODE REVIEW

File: en-US/features/authentication.md [hunk 1]
```diff
@@ -127,3 +127,37 @@ This option allow Gogs to login in your SMTP host as Gogs user. To configure thi
   
 - This authentication is activate
   - Enable or disable this auth
+
+##Freeipa
```
Comment (Unknwon) [inline en-US/features/authentication.md]: A space after `#`.

File: en-US/features/authentication.md [hunk 2]
```diff
@@ -127,3 +127,37 @@ This option allow Gogs to login in your SMTP host as Gogs user. To configure thi
...
+
+- In order to login to the Gogs using FreeIPA credentials, you need to create a bind account for GOGS to use:
```
Comment (Unknwon) [inline en-US/features/authentication.md]: `Gogs`

File: en-US/features/authentication.md [hunk 3]
```diff
@@ -127,3 +127,37 @@ This option allow Gogs to login in your SMTP host as Gogs user. To configure thi
...
+
+-  On the FreeIPA server, create a gogs.ldif file, replacing dc=example,dc=com with your DN, and providing an appropriately secure password:
```
Comment (Unknwon) [inline en-US/features/authentication.md]:
```
`gogs.ldif`
```

**Comment by Karen09** [comment]
Hi.
Have you any other questions ?

Regards
Karen

**Comment by Unknwon** [comment]
Hmm... seems having conflicts...

**Comment by kalinux** [comment]
Hi,

I can`t see where is exactly conflicts ? could you give more info please ?

Regards
Karen

**Comment by Unknwon** [comment]
I don't know exactly, but GitHub says so... you should pull from upstream and merge the change, then you will find out where are the conflicts.

**Comment by kalinux** [comment]
Something is wrong . Let me create one more time pull request

**Pull request closed without merging**